torch/distributed/_state_dict_utils.py

```diff
@@ -296,6 +321,46 @@ def _offload_state_dict_to_cpu(
     return ret
 
 
+def _copy_state_dict(
+    state_dict: Dict[str, Any],
+    copy_state_dict: Dict[str, Any],
+    non_blocking: bool = False,
+):
```

Comment:
Since we move the copy function as a standalone API, we should also remove `cpu_offload_state_dict` and `cpu_offload_state_dict_sync` from `_offload_state_dict_to_cpu`